src/main/routes/routes.ts

```diff
@@ -62,9 +62,11 @@ export default function(app: Application): void {
     regenerateSession);
   app.get('/login', passport.authenticate(authType, { failureRedirect: '/'}),
     regenerateSession);
-  app.get('/subscription-add', ensureAuthenticated, app.locals.container.cradle.subscriptionAddController.get);
-  app.post('/subscription-add', ensureAuthenticated, app.locals.container.cradle.subscriptionAddController.post);
-  app.get('/status-description', app.locals.container.cradle.statusDescriptionController.get);
+
+  app.get('/subscription-add', app.locals.container.cradle.subscriptionAddController.get);
+  app.post('/subscription-add', app.locals.container.cradle.subscriptionAddController.post);
```

Comment:
Should 'ensureAuthenticated' not still be required for these screens?

Reply:
added again. I mistakenly removed while fixing the code conflict.